libs/application/templates/driving-license/src/forms/draft/subSectionHealthDeclaration.ts

```diff
@@ -124,6 +125,14 @@ export const subSectionHealthDeclaration = buildSubSection({
             label: m.healthDeclaration10,
           },
         ),
+        buildAlertMessageField({
+          id: 'healthDeclaration.contactGlassesMismatch',
+          title: '',
+          message: m.alertHealthDeclarationGlassesMismatch,
+          alertType: 'warning',
+          condition: (answers) =>
+            (answers.healthDeclaration as any)?.contactGlassesMismatch,
```

Comment:
I literally do not remember how this part of typescript works, but could this be replaced with unknown instead of any?

Reply:
no, it does not compile